### PR TITLE
Perseus Questions

### PR DESCRIPTION
**Pull request by jayoshih**
Added in perseus questions for raw data

### CODE REVIEW

File: ricecooker/classes/nodes.py [hunk 1]
```diff
@@ -5,12 +5,46 @@
...
 from io import BytesIO
+from PIL import Image
+from ricecooker import config
 from le_utils.constants import content_kinds,file_formats, format_presets, licenses, exercises
-from ricecooker.exceptions import UnknownQuestionTypeError, InvalidInputAnswerException
 
-def guess_content_kind(files, questions=[]):
+def download_image(path):
+    filename, original_filename, path, file_size = download_file(path, '.{}'.format(file_formats.PNG))
+    return '![]' + exercises.IMG_FORMAT.format(filename), filename, original_filename, path, file_size
```
Comment (aronasorman) [inline ricecooker/classes/nodes.py]: What does this `![]` syntax mean? For perseus?

File: ricecooker/classes/nodes.py [hunk 2]
```diff
@@ -127,16 +172,21 @@ class Node(TreeModel):
...
-        self.id = id
-        self.title = title
-        self.description = description
-        self.author = author
-        self.license = license
-        self.children = []
+    def __init__(self, *args, **kwargs):
+        self.id = args[0]
+        self.title = args[1]
+        self.description = "" if 'description' not in kwargs else kwargs['description']
```
Comment (aronasorman) [inline ricecooker/classes/nodes.py]: Can be rewritten to:

`self.description = kwargs.get('description', "")`

File: ricecooker/classes/nodes.py [hunk 3]
```diff
@@ -127,16 +172,21 @@ class Node(TreeModel):
...
+        self.author = kwargs['author']
+        self.license = None if 'license' not in kwargs else kwargs['license']
```
Comment (aronasorman) [inline ricecooker/classes/nodes.py]: Rewrite to `kwargs.get`

File: ricecooker/classes/nodes.py (hunk 1 above)
Comment (jayoshih) [inline ricecooker/classes/nodes.py]: Removed in next PR

File: ricecooker/classes/nodes.py (hunk 2 above)
Comment (jayoshih) [inline ricecooker/classes/nodes.py]: Fixed!

**Pull request merged**